src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_UnmanagedCallersOnly.vb

```diff
@@ -0,0 +1,404 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Roslyn.Test.Utilities
+Imports Roslyn.Test.Utilities.TestMetadata
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+    Public Class AttributeTests_UnmanagedCallersOnly
```

Comment:
I am not sure if Emit unit-tests is the right place for these tests. Are they actually validating emit behavior? Perhaps the file should be moved to Symbols tests?